types/php-serialize/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for php-serialize 3.0
+// Project: https://github.com/steelbrain/php-serialize
+// Definitions by: Changhui Lee <https://github.com/blurfx>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.7
+
+export function isSerialized(givenItem: any, strict: any): any;
```

Comment:
[`strict` is optional](https://github.com/steelbrain/php-serialize/blob/master/src/isSerialized.js#L8).

```suggestion
export function isSerialized(givenItem: any, strict?: any): any;
```